frontend/pages/hosts/HostDetailsPage/EmptySoftware/EmptySoftware.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const baseClass = "empty-software";
+
+const EmptySoftware = (): JSX.Element => {
+  return (
+    <div className={`${baseClass}`}>
+      <div className={`${baseClass}__inner`}>
+        <div className={`${baseClass}__empty-filter-results`}>
+          <h1>Oops! No software matched your search criteria</h1>
```

Comment:
cc @noahtalerman maybe we'd like different messaging here? Possibly just remove the "Oops!"?

Reply:
Yes, let's please use "No software match your current search criteria."